README.md

```diff
@@ -4,6 +4,13 @@ swiftly is a CLI tool for installing, managing, and switching between [Swift](ht
 
 Ongoing maintenance and stewardship of this project is led by the [SSWG](https://www.swift.org/sswg/).
 
+### Installation
+
+Just Run:
```

Comment:
Maybe make this a little more formal
```suggestion
To download Swiftly and install Swift, run the following in your terminal, then follow the on-screen instructions.
```
I ripped that from the rust installation screen

Reply:
I also ripped that "Just Run" from rustlings README 😅

Reply:
Yeah generally let's avoid "simply" "just" etc, these steps may not be super trivial to some newcomers :)